packages/paypal-commerce-integration/src/paypal-commerce-integration-service.ts

```diff
@@ -158,13 +158,18 @@ export default class PayPalCommerceIntegrationService {
         });
     }
 
-    async submitPayment(methodId: string, orderId: string): Promise<void> {
+    async submitPayment(
+        methodId: string,
+        orderId: string,
+        additionalData?: { [key: string]: unknown },
+    ): Promise<void> {
```

Comment:
Sorry about returning to this method, but I have a better idea:
Instead of using this particular method you can use 
`await this.paymentIntegrationService.submitPayment({ methodId, paymentData });`
in a ratepay strategy to avoid adding extra logic here. So your implementation will be isolated into your own strategy. What do you think?

Reply:
fixed

Reply:
@serhii-tkachenko Yes

Reply:
Yes, i did like this